op-acceptance-tests/tests/sync_tester/init_test.go

```diff
@@ -5,10 +5,15 @@ import (
 
 	"github.com/ethereum-optimism/optimism/op-devstack/compat"
 	"github.com/ethereum-optimism/optimism/op-devstack/presets"
+	stconf "github.com/ethereum-optimism/optimism/op-sync-tester/synctester/backend/config"
 )
 
 func TestMain(m *testing.M) {
-	presets.DoMain(m, presets.WithMinimalWithSyncTester(),
+	presets.DoMain(m, presets.WithMinimalWithSyncTester(&stconf.TargetBlocks{
```

Comment:
Can we reuse 
```go
type FCUState struct {
	Latest    uint64
	Safe      uint64
	Finalized uint64
}
```

Also I feel like start block makes more sense, not target block for the naming.